### PR TITLE
2018 04 22b

### PR DESCRIPTION
**Pull request by Gofannon**
## Problem
- Requirements for https://github.com/YunoHost-Apps/dokuwiki_ynh/issues/56

## Solution
- upgrade to lastest upstream version
- Fix template for PR/MR
- Add changelog
- Fix READMEs

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Kay0u
- [x] **Approval (LGTM)** : Kay0u
- [x] **Approval (LGTM)** : Maniack C
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/dokuwiki_ynh%20PR58/)  

When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by Gofannon** [comment]
Tested by upgrading from latest stable version

File: pull_request_template.md [hunk 1]
```diff
@@ -13,12 +13,10 @@
...
-*Minor decision*
-- **Upgrade previous version** : 
 - [ ] **Code review** : 
-- [ ] **Approval (LGTM)** : 
-- [ ] **Approval (LGTM)** : 
+- [ ] **Approval (LGTM)** :  
+*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
 - **CI succeeded** : 
-[![Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/dokuwiki_ynh%20PR-NUM-/badge/icon)](https://ci-apps-hq.yunohost.org/jenkins/job/dokuwiki_ynh%20PR-NUM-/)  
+[![Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/APP_ynh%20PR-NUM-/badge/icon)](https://ci-apps-hq.yunohost.org/jenkins/job/APP_ynh%20PR-NUM-/)
```
Comment (kay0u) [inline pull_request_template.md]: Uuh dokuwiki was good here

File: manifest.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
         "es": "DokuWiki es un sistema de Wiki de uso sencillicimo y compatible con los estándares.",
         "it": "DokuWiki è un Wiki aderente agli standard, semplice da usare, finalizzato principalmente alla creazione di documentazione di qualsiasi tipo."
     },
-    "version": "2018-04-22a~ynh3",
+    "version": "2018-04-22b~ynh4",
```
Comment (kay0u) [inline manifest.json]:
```suggestion
    "version": "2018-04-22b~ynh1",
```
If the app version is changed, set the package version to 1 ;-)

File: pull_request_template.md (hunk 1 above)
Comment (Gofannon) [reply pull_request_template.md]: failed copy pasta

File: manifest.json (hunk 2 above)
Comment (Gofannon) [reply manifest.json]: I wasn't aware of that. I thought that the package version should "never change", only be increased. Well then :)

**Comment by Gofannon** [comment]
bump

**Pull request merged**